primitives/teerex/src/lib.rs

```diff
@@ -143,10 +143,16 @@ impl TcbVersionStatus {
 
 	pub fn verify_examinee(&self, examinee: &TcbVersionStatus) -> bool {
 		for (v, r) in self.cpusvn.iter().zip(examinee.cpusvn.iter()) {
+			log::debug!("teerex: verify_examinee: v={:#?},r={:#?}", v, r);
```

Comment:
Thanks a lot for all these extra logs. However, I am not fond of `teerex:`, it is essentially redundant information as the logger does already contain the rust module path. But we could add a `target: TEEREX` as an argument, where teerex should be a const ;)

Reply:
I did not know about this, handy feature! Rewrote all of them to use it, thanks for the early review! :D :100: